analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/UsingNonstandardCryptographyBase.cs

```diff
@@ -44,49 +44,31 @@ public abstract class UsingNonstandardCryptographyBase<TSyntaxKind, TTypeDeclara
                 KnownType.System_Security_Cryptography_DeriveBytes,
                 KnownType.System_Security_Cryptography_HashAlgorithm,
                 KnownType.System_Security_Cryptography_ICryptoTransform,
-                KnownType.System_Security_Cryptography_SymmetricAlgorithm
-            );
-
-        protected abstract GeneratedCodeRecognizer GeneratedCodeRecognizer { get; }
+                KnownType.System_Security_Cryptography_SymmetricAlgorithm);
```

Comment:
Question: Who is taking care that such lists are up to date?

Reply:
Nobody and everybody. This rule is under the security umbrella so AppSec bubble might specify an update. Or we can. Or community can.